src/illus/nocreations.svg

```diff
@@ -0,0 +1,14 @@
+<svg width="176" height="128" viewBox="0 0 176 128" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
`nocreations` is vague name. To me it looks like a illustration of the browser, so maybe `browser.svg`? @peratik what do you think?